etl/steps/data/garden/papers/2022-11-04/riley_2005/meta.yml

```diff
@@ -5,13 +5,9 @@ dataset:
   description: |
     Historians and demographers have gone to considerable trouble to reconstruct life expectancy in the past in individual countries.
 
-    This overview collects information from a large body of that work and links estimates for historical populations to those provided by
-    the United Nations, the World Bank, and other sources for 1950–2001. The result is a picture of regional and global life expectancy 
-    at birth for selected years from 1800 to 2001. The bibliography of more than 700 sources is published separately on the web.
+    This overview collects information from a large body of that work and links estimates for historical populations to those provided by the United Nations, the World Bank, and other sources for 1950–2001. The result is a picture of regional and global life expectancy  at birth for selected years from 1800 to 2001. The bibliography of more than 700 sources is published separately on the web.
```

Comment:
"large body of that work" typo?

Reply:
Not sure I follow here. I copied the abstract from the paper (see https://onlinelibrary.wiley.com/doi/10.1111/j.1728-4457.2005.00083.x)